src/components/CellWithPopover/CellWithPopover.scss

```diff
@@ -27,7 +25,10 @@
         }
     }
     &__children-wrapper {
+        overflow: hidden;
+
         cursor: pointer;
+        text-overflow: ellipsis;
```

Comment:
The text should be trimmed from the left

<img width="288" height="409" alt="Screenshot 2025-09-01 at 14 46 34" src="https://github.com/user-attachments/assets/08e062d0-9f9c-4125-bb22-29bf70267ce1" />


The problem here in the `className` prop, that is passed from https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/containers/Storage/PaginatedStorageGroupsTable/columns/columns.tsx#L44, by due to some changes in last uikit is not applied. It seems, the problem with poolName appeared after uikit update

Reply:
If is fixed by adding `className` prop to `children-wrapper`
```
<div className={b('children-wrapper', {'full-width': fullWidth}, className)}>
    {children}
</div>
```